providers/flagd/pom.xml

```diff
@@ -41,22 +41,22 @@
         <dependency>
             <groupId>io.grpc</groupId>
             <artifactId>grpc-netty</artifactId>
-            <version>1.57.0</version>
+            <version>1.57.1</version>
         </dependency>
 
         <dependency>
             <!-- we only support unix sockets on linux, via epoll native lib -->
             <groupId>io.netty</groupId>
             <artifactId>netty-transport-native-epoll</artifactId>
-            <version>4.1.96.Final</version>
+            <version>4.1.95.Final</version>
```

Comment:
if you think `.96` is bugged, let's be careful updating it again.

I will prioritize this [next](https://github.com/open-feature/java-sdk-contrib/issues/380), so we can have some e2e tests here.

Reply:
Root cause was introduction of a static block validation. I opened an issue for netty - https://github.com/netty/netty/issues/13523